Bottom: 2ed6bf68b41ebc55a3d9658b236f913f150b3668
Top:    4361dedde73463703d1296ae659a860127191390
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-05-08 20:21:54 +0100

Refresh of shiny-3d-eye-candy-0

---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index c36cc43..db33d19 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -107,8 +107,6 @@ ghid_set_layer (const char *name, int group, int empty)
     }
   }
 
-#define SUBCOMPOSITE_LAYERS
-#ifdef SUBCOMPOSITE_LAYERS
   glEnable (GL_STENCIL_TEST);                                 /* Enable Stencil test */
   glStencilOp (GL_KEEP, GL_KEEP, GL_REPLACE);                 /* Stencil pass => replace stencil value (with 1) */
 
@@ -1055,7 +1053,7 @@ ghid_drawing_area_expose_cb (GtkWidget *widget,
   /* Just prod the drawing code so the current depth gets set to
      the right value for the layer we are editing */
   gui->set_layer (NULL, GetLayerGroupNumberByNumber (INDEXOFCURRENT), 0);
-  gui->set_layer (NULL, SL_FINISHED, 0);
+  gui->end_layer ();
 
   ghid_draw_grid (&region);
